portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,18 +15,41 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import com.google.gson.Gson;
+import java.util.ArrayList;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content. TODO: modify this file to handle messages data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  
+  private ArrayList<String> messages;
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Areeta!</h1>");
+
+    // Convert the messages to JSON
+    messages = new ArrayList<>();
+    messages.add("boop bop beep");
+    messages.add("bopity beep boop");
+    messages.add("beep boop bop");
+    String json = convertToJsonUsingGson(messages);
+
+    // Send the JSON as the response
```

Comment:
It's usually recommended to end comments with a period.

Reply:
Gotcha, thank you!